1-exercises/F-for-of-loop/exercise.js

```diff
@@ -4,13 +4,20 @@
 
 // TODO Use a for-of loop to output each of the tube stations below.
 let tubeStations = [
-    "Aldgate",
-    "Baker Street",
-    "Picadilly Circus",
-    "Oxford Street",
-    "Tottenham Court Road"
+  "Aldgate",
+  "Baker Street",
+  "Picadilly Circus",
+  "Oxford Street",
+  "Tottenham Court Road",
 ];
 
+for (let i = 0; i < tubeStations.length; i++) {
```

Comment:
Nice work!.
This works perfectly fine, but the requirement is to use a for-of loop.
See [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) and [W3Schools](https://www.w3schools.com/js/js_loop_forof.asp) for more information.